docs/research/user-personas/user-persona-digital-safety-conscious-parent-P01.md

```diff
@@ -0,0 +1,89 @@
+# User Persona – Digital Safety-Conscious Parent (P01)
+
+## Basic Profile
+| Attribute | Description |
+|-----------|-------------|
+| Name | Digital Safety-Conscious Parent (P01) |
+| Age | Late 30s–40s |
+| School Level | University-educated (inferred from structured digital management & MOE familiarity) |
+| Tech Proficiency | Medium–High (Apple Family, apps, tracking tools, Parents Gateway) |
+| Location / Community | Singapore, with children in MOE school system |
```

Comment:
I think worth noting the race. Cultural differences might be note worthy.

Reply:
Added to this file and to notes template